common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -662,6 +662,18 @@ def create_input_element(self, **kwargs):
         return etree.Element("javascriptinput")
 
 
+class JSInputXMLFactory(CustomResponseXMLFactory):
```

Comment:
Used this stub factory for an a11y bok choy test. Unfortunately jsinput had basically no test coverage previously (certainly no server-side coverage). Creating a fully functional jsinput test problem seemed beyond the scope of this story.